File: packages/insomnia-httpsnippet/test/fixtures/output/csharp/restsharp/application-form-encoded.cs
var client = new RestClient("http://mockbin.com/har");
var request = new RestRequest(Method.POST);
request.AddHeader("content-type", "application/x-www-form-urlencoded");
request.AddParameter("application/x-www-form-urlencoded", "foo=bar&hello=world", ParameterType.RequestBody);
IRestResponse response = client.Execute(request);

File: packages/insomnia-httpsnippet/test/fixtures/output/node/request/application-form-encoded.js
var request = require('request');

var options = {
  method: 'POST',
  url: 'http://mockbin.com/har',
  headers: { 'content-type': 'application/x-www-form-urlencoded' },
  form: { foo: 'bar', hello: 'world' },
};

request(options, function(error, response, body) {
  if (error) throw new Error(error);

  console.log(body);
});
